src/sentry/api/endpoints/organization_profiling_profiles.py

```diff
@@ -118,3 +128,36 @@ def get(self, request: Request, organization: Organization) -> StreamingHttpResp
             kwargs["headers"] = {"Accept-Encoding": request.headers.get("Accept-Encoding")}
 
         return proxy_profiling_service("GET", f"/organizations/{organization.id}/filters", **kwargs)
+
+
+class OrganizationProfilingStatsEndpoint(OrganizationProfilingBaseEndpoint):
```

Comment:
private label?

Reply:
it inherits `private = True` from the base endpoint

Reply:
Okay, so as long as we don't add a public endpoint to the base, we should be fine.